_posts/2022-02-01-engineering-product-design.md

```diff
@@ -0,0 +1,73 @@
+* Apps
+* IT Services
+
+## Traffic 
+The teams inside Traffic are responsible for bringing the users to our platform. Our projects include keywords like SEO, landing pages or user generated content. We have our hands and minds on all those pages where our users land at HolidayCheck when they start planning their holidays. The teams are all Fullstack, being Javascript, Typescript, React, NodeJS, GraphQL, ElasticSearch or Java some of the technologies they use. 
+
+Our latest successful projects include: 
+* We help our users know which location is the right one for them, like [Greece](https://www.holidaycheck.de/urlaub/griechenland) 
+* We keep them informed about the latest [travel restrictions](https://www.holidaycheck.de/sommerurlaub) 
+* We show them the first [prices](https://www.holidaycheck.de/fruehbucher), when they are still undecided  
+
+## E-Commerce 
+At the Ecom teams, the focus lies on helping users on our platform find the right hotel, the best offers and deals to have an unforgettable vacation. We continuously improve the booking funnel, develop services connecting to our external partners or develop the newest features related to the HolidayCheck Club. The teams are both Fullstack and Backend focused, and the technologies include: Javascript, React, NodeJS, Scala. 
+
+Our greatest late additions are: 
+* Flex offers, where we are proud to have changed the travel industry by being the first ones to offer such a product 
+* New and optimised hotel pages, as [Dana Beach Resort](https://www.holidaycheck.de/hi/dana-beach-resort/1aa4c4ad-f9ea-3367-a163-8a3a6884d450) 
+* Premium offers and deals you can find all around the web 
+
+## Sales 
+At the Sales team we combine our massive reach and deep understanding of our markets to build innovative advertising products that generate the most effective and meaningful touchpoints between our Urlaubers and our leasure travel industry partners. We also make vacations better for our Urlaubers by developing tools to make relevant insights, benchmarks and reports easily accessible and actionable for our partners. Our team is full stack and we work with JavaScript, especially Node/Express and React. 
+
+Our latest projects include: 
+* Building out the HolidayCheck Business Center for partners to navigate the HolidayCheck universe from one cockpit 
+* Making our advertising products more accessible and easier to use through self-booking 
+* Introducing auction/bidding to our advertising products for better price-discovery 
+
+## Data/ML 
+The Data and ML team is responsible for various projects, both on the platform and for generating internal insights. Our data lake is hosted on AWS and runs over 800 batch jobs every night, storing every offer that we see on our website and aggregating every interaction the user has with us. This allows us at HC to have all relevant data in one location and easily consumable. Based on this asset we have been building additional value with these projects: 
```

Comment:
there are easily more than 1000 jobs

Reply:
"aggregating every interaction the user has with us" might be too specific again raising concerns on privacy. Can we change it to "processing the huge amounts of data giving customised experience to the user"